strategies/affected/src/test/java/org/arquillian/smart/testing/strategies/affected/fakeproject/test/YTest.java

```diff
@@ -0,0 +1,19 @@
+package org.arquillian.smart.testing.strategies.affected.fakeproject.test;
+
+import org.arquillian.smart.testing.strategies.affected.Tests;
+import org.arquillian.smart.testing.strategies.affected.fakeproject.main.superbiz.Alone;
+import org.junit.Ignore;
+import org.junit.Test;
+
+// Test ignored because it is a test that is used to in tests and not to be run as real test by test runner
```

Comment:
I know what you are trying to say in this sentence, but for a person not familiar with test-bed and how we tests our test selection this sentence would be rather confusing - can you make it more digestable? :)

You can also put it as a part of `@Ignore`.